source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -87,6 +87,8 @@ Once you create a CDN service instance, you cannot update or delete the instance
 
 1. Create one CNAME record for each subdomain using the DNS information. This directs traffic to your service.
 
+    You should not create CNAME records until your cdn-route service instance has been created. Run `cf services SERVICE_INSTANCE` to check the status of your service instance. If the service has been created, the service status will say `Create succeeded`.
```

Comment:
It should say `cf service SERVICE_INSTANCE`.

Reply:
oh i'm a silly man, thanks